src/SprykerEco/Yves/Computop/Mapper/Init/PostPlace/CreditCardMapper.php

```diff
@@ -251,4 +252,35 @@ protected function expandCreditCardPaymentWithCredentialOnFile(
 
         return $computopCreditCardPaymentTransfer;
     }
+
+    /**
+     * @param \Generated\Shared\Transfer\AddressTransfer $addressTransfer
+     * @param \Generated\Shared\Transfer\ComputopConsumerTransfer $computopConsumerTransfer
+     *
+     * @return \Generated\Shared\Transfer\ComputopConsumerTransfer
+     */
+    protected function mapAddressTransferToConsumerTransfer(
+        AddressTransfer $addressTransfer,
+        ComputopConsumerTransfer $computopConsumerTransfer
+    ): ComputopConsumerTransfer {
+        $computopConsumerTransfer->fromArray($addressTransfer->toArray(), true);
+        if ($computopConsumerTransfer->getSalutation()) {
+            $computopConsumerTransfer->setSalutation($this->getAcceptedSalutation($computopConsumerTransfer->getSalutation()));
+        }
+
+        return $computopConsumerTransfer;
+    }
+
+    /**
+     * @param string $salutation
+     *
+     * @return string
+     */
+    protected function getAcceptedSalutation(string $salutation): string
```

Comment:
is it really `accepted`? naming looks strange

Reply:
Fixed